src/lib/litegraph/src/LGraphCanvas.ts

```diff
@@ -688,8 +688,8 @@ export class LGraphCanvas
 
   /** If true, enable drag zoom. Ctrl+Shift+Drag Up/Down: zoom canvas. */
   dragZoomEnabled: boolean = false
-  /** The start position of the drag zoom. */
-  #dragZoomStart: { pos: Point; scale: number } | null = null
+  /** The start position of the drag zoom and original read-only state. */
+  #dragZoomStart: { pos: Point; scale: number; readOnly: boolean } | null = null
```

Comment:
Want to make this typescript private while you're in here?
```suggestion
  private dragZoomStart: { pos: Point; scale: number; readOnly: boolean } | null = null
```